.eslintrc

```diff
@@ -153,7 +153,7 @@ rules:
   max-len: 0
   max-nested-callbacks: 0
   max-params: 0
-  max-statements: [2, 50]
+  max-statements: [2, 60]
```

Comment:
Just 60? I expected to see an order of magnitude ;-)

`max-statements: [2, 500]`